README.md

```diff
@@ -559,6 +564,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), 2017.
 *[HATS: Histograms of Averaged Time Surfaces for Robust Event-based Object Classification](https://arxiv.org/abs/1803.07913)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR) 2018. 
     - [N-CARS Dataset](#ncars_dataset): A large real-world event-based dataset for car classification.
+- <a name="Zhu2018rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,  
```

Comment:
This entry is duplicated. Please link to the primary entry in the optical flow section. See the example from Bardow's entry, which is also in more than one category.

Reply:
Updated